### ISSUE TITLE
False Positive: Layout/ExtraSpacing

### ISSUE THREAD

**Issue by armin86er** [rubocop-hq/rubocop#7579]
--------

## Expected behavior

With the configuration: 

```yml
Layout/ExtraSpacing:
  ForceEqualSignAlignment: true
```
I wouldn't expect any offense with the following code:

```ruby
# frozen_string_literal: true

def foo
  bar = 42
  bar.methodcall
  foobar = 24
  foobar.methodcall
end
```

## Actual behavior

```
Offenses:

C: Layout/ExtraSpacing: = is not aligned with the preceding assignment.
  foobar = 24
         ^
```

The changed and not offended code looks like this:

```ruby
# frozen_string_literal: true

def foo
  bar    = 42
  bar.methodcall
  foobar = 24
  foobar.methodcall
end
```

## Steps to reproduce the problem

Simply the ruby code and the configuration above, processed by rubocop with versions greater or equal 0.74 (including 0.78, which is actual the newest release). 

I think it's caused by changes due to #5265.

## RuboCop version
```
$ [bundle exec] rubocop -V
0.74.0 (using Parser 2.6.0.0, running on ruby 2.6.5 x86_64-linux)
```

**Comment by buehmann** [comment]
I think this is the expected behaviour, which changed in https://github.com/rubocop-hq/rubocop/pull/7211 if I remember correctly. The cop now demands that equal signs be aligned within a "paragraph". That means this would also be acceptable code:

```ruby
# frozen_string_literal: true

def foo
  bar = 42
  bar.methodcall

  foobar = 24
  foobar.methodcall
end
```

**Comment by armin86er** [comment]
Ah, yes it seems like the expected behaviour. I'll close the issue.
